bravado/mapping/schema.py

```diff
@@ -10,6 +11,14 @@
     'null': (),
 }
 
+SWAGGER_PRIMITIVES = (
+    'integer',
+    'number',
+    'string',
+    'boolean',
+    'null'     # TODO: Do we need this?
```

Comment:
according to the spec, this is correct

https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md#data-types

references

http://json-schema.org/latest/json-schema-core.html#anchor8

Although I guess swagger special cases "array" and "object", and "file" is missing, but I guess that's a known deficiency right now? Do we have an issue open for "add file support for bravado" ?